controllers/datasciencecluster/datasciencecluster_controller.go

```diff
@@ -496,10 +407,8 @@ func (r *DataScienceClusterReconciler) SetupWithManager(ctx context.Context, mgr
 		Owns(&componentsv1.TrainingOperator{}).
 		Owns(&componentsv1.DataSciencePipelines{}).
 		Owns(&componentsv1.Kserve{}).
-		Owns(
-			&corev1.ServiceAccount{},
-			builder.WithPredicates(saPredicates),
-		).
+		Owns(&componentsv1.ModelMeshServing{}).
+		Owns(&componentsv1.ModelController{}).
```

Comment:
We also need to watch the status for all the components so that we can update DSC's status when a component status change so I guess we either need a custom predicate or we should remove the global `WithEventFilter` below.

Reply:
i think it is about this one https://issues.redhat.com/browse/RHOAIENG-15042?

Reply:
It is, but since you are removing a bunch of "own" and "predicates" that should probably be part of the [RHOAIENG-15042](https://issues.redhat.com/browse/RHOAIENG-15042), I tough you were including some of that bits too.